internal/ast/utilities.go

```diff
@@ -2648,6 +2658,10 @@ func nodeContainsPosition(node *Node, position int) bool {
 	return node.Kind >= KindFirstNode && node.Pos() <= position && (position < node.End() || position == node.End() && node.Kind == KindEndOfFile)
 }
 
+func NodeRangeContainsPosition(node *Node, pos int) bool {
```

Comment:
All uses of this can just be `node.Loc.ContainsInclusive(pos)` - our text ranges have methods now.